src/oid4vci/error.rs

```diff
@@ -11,31 +13,37 @@ pub enum Oid4vciError {
     #[error("Serde error")]
     SerdeJsonError(String),
 
-    #[error("HTTP request error: {0}")]
+    #[error("HTTP request error: {_0}")]
     RequestError(String),
 
     #[error("Unsupported grant type")]
     UnsupportedGrantType,
 
-    #[error("Invalid session: {0}")]
+    #[error("Invalid session: {_0}")]
     InvalidSession(String),
 
-    #[error("Invalid parameter: {0}")]
+    #[error("Invalid parameter: {_0}")]
     InvalidParameter(String),
 
-    #[error("Failed to acquire lock for {0}")]
+    #[error("Failed to acquire lock for {_0}")]
     LockError(String),
 
     #[error("{vp_request}")]
     VpRequestRequired { vp_request: serde_json::Value },
 
+    #[error("ProofValidationError: {_0}")]
+    ProofValidationError(#[from] ProofValidationError),
+
+    #[error("DecodeError: {_0}")]
+    DecodeError(#[from] DecodeError),
+
     #[error("{_0}")]
     DidError(#[from] DidError),
 
     #[error("{_0}")]
     ContextMapError(#[from] FromContextMapError),
 
-    #[error("{0}")]
+    #[error("{_0}")]
```

Comment:
What does this change do?

Reply:
Not sure what the difference between `{0}` and `{_0}` is.
I changed them all to be consistent inside the enum.